openff/nagl_models/openff_nagl_models.py

```diff
@@ -139,9 +146,76 @@ def list_available_nagl_models() -> List[str]:
         ['/Users/lily/pydev/openff-nagl-models/openff/nagl_models/models/openff-gnn-am1bcc-0.0.1-alpha.1.pt']
 
     """
-    import glob
     model_paths = load_nagl_model_directory_entry_points()
     model_files = []
     for path in model_paths:
         model_files.extend(glob.glob(os.path.join(path, "*.pt")))
     return sorted([os.path.abspath(f) for f in model_files])
+
+
+def _get_latest_model(model_type: str, production_only: bool = False) -> str:
```

Comment:
I'm trying to think of a better middle ground here where the warning isn't necessary. 

Would it be more reasonable for this method to be more "get an ordered list of a given model type"?

In that case you guarantee a release ordered list, but you don't necessarily prescribe anything to the method user - it just happens to be ordered.

Reply:
Thanks for having a look @IAlibay! That's a great idea -- something like a `get_models_by_type` method where the results are sorted by version is much less worrying and makes sense. I don't personally anticipate the file names changing much but it's hard to guarantee that very far in the future in case we do go in a different direction -- having this method in the library at least leaves the way open to defining new naming schemes, etc., with minimal disruption downstream.